addons/ctrl-click-run-scripts/addon.json

```diff
@@ -25,7 +29,20 @@
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Execute on double click",
```

Comment:
Does this need to be a setting? How often will someone accidentally double-click something? I think having multiple ways to activate it without any settings is fine and the middle-click popup already does it. If this is done then I think it would make sense to also rename the addon to use double-click.

Reply:
One way making a setting for double-clicking to run is that it could make the double-click feature more discoverable.